stylesheets/abstracts/_mixins.scss

```diff
@@ -32,5 +7,40 @@
   }
 }
 
+@mixin display {
+  display: flex;
+  justify-content: space-between;
+}
+
+@mixin typography($size, $lineHeight) {
+  font-size: $size;
+  line-height: $lineHeight;
+}
+
+@mixin typography-style-parag($weight, $size, $lineHeight) {
+  @include typography($size, $lineHeight);
+  color: $paragraph-color;
+  font-weight: $weight;
+  font-family: $text-font;
+}
+
+@mixin typography-weight600 ($color, $size, $lineHeight) {
+  color: $color;
+  font-weight: 600;
+  @include typography($size, $lineHeight);
+  font-family: $text-font;
+}
 
+@mixin typography-weight700 ($size, $lineHeight) {
+  color: $brand-color;
+  font-weight: 700;
+  @include typography($size, $lineHeight);
+  font-family: $text-font;
+}
```

Comment:
These mixing are identical except for font-weight.
Add font-weight as a mixing parameter.
Use kebab-case for ALL variables names.